spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -551,7 +551,7 @@ trait DeltaSQLConfBase {
   /////////////////////////////////////////////
 
   val MANAGED_COMMIT_DDB_AWS_CREDENTIALS_PROVIDER_NAME =
-    buildConf("managedCommits.commitOwner.ddb.awsCredentialsProviderName")
+    buildConf("managedCommit.commitOwner.ddb.awsCredentialsProviderName")
```

Comment:
Could we make it more explicit: `managedCommit.dynamoDBCommitOwner.awsCredentialsProviderName` or `managedCommit.dynamoDbCommitOwner.awsCredentialsProviderName`?

Reply:
How about managedCommit.commitOwner.dynamodb.awsCredentialsProviderName